service/certconfig/v2/resources/vaultrole/resource.go

```diff
@@ -11,7 +11,7 @@ const (
 	// AllowSubdomains defines whether to allow the generated root CA of the PKI
 	// backend to allow sub domains as common names.
 	AllowSubdomains = true
-	Name            = "vaultrolev1"
+	Name            = "vaultrolev2"
```

Comment:
Note the name changes here to match the right version. The other resource names stay the same since they already have been v2.